Voycar.Api.Web.Tests.Integration/Users/Post_ResetPassword/Tests.cs

```diff
@@ -0,0 +1,168 @@
+    }
+
+
+    [Fact]
+    public async Task Post_Request_AsEmployee_ReturnsOk_And_ResetPassword()
+    {
+        // Arrange
+        const string email = "employee@test.de";
+        var employeeClient = await ClientFactory.CreateEmployeeClient(this._app, this.Context, email, "password");
+        var request = new R.Request
+        {
+            PasswordResetToken = "2CE2B362-926C-43A9-87CE-5DADE92E1DBF", Password = "newPassword"
+        };
+
+        var userInDb = this.Context.Users.First(user => user.Email == email);
+        userInDb.PasswordResetToken = "2CE2B362-926C-43A9-87CE-5DADE92E1DBF";
+        userInDb.ResetTokenExpires = DateTime.UtcNow.AddHours(3);
+
+        await this.Context.SaveChangesAsync();
+
+        // Act
+        var firstHttpResponse = await employeeClient.POSTAsync<R.Endpoint, R.Request>(request);
+        var secondHttpResponse = await employeeClient
+            .POSTAsync<L.Endpoint, L.Request>(new L.Request { Email = email, Password = request.Password });
+
+        // Assert
+        userInDb.Should().NotBeNull();
+
+        firstHttpResponse.StatusCode.Should().Be(HttpStatusCode.OK);
+        secondHttpResponse.StatusCode.Should().Be(HttpStatusCode.OK);
+    }
+
+
+    [Fact]
+    public async Task Post_Request_AsAdmin_ReturnsOk_And_ResetPassword()
+    {
+        // Arrange
+        const string email = "admin@test.de";
+        var adminClient = await ClientFactory.CreateAdminClient(this._app, this.Context, email, "password");
+        var request = new R.Request
+        {
+            PasswordResetToken = "799A354C-7AD6-4305-8F34-72DC4ABFDD2F", Password = "newPassword"
+        };
+
+        var userInDb = this.Context.Users.First(user => user.Email == email);
+        userInDb.PasswordResetToken = "799A354C-7AD6-4305-8F34-72DC4ABFDD2F";
+        userInDb.ResetTokenExpires = DateTime.UtcNow.AddHours(3);
+
+        await this.Context.SaveChangesAsync();
+
+        // Act
+        var firstHttpResponse = await adminClient.POSTAsync<R.Endpoint, R.Request>(request);
+        var secondHttpResponse = await adminClient
+            .POSTAsync<L.Endpoint, L.Request>(new L.Request { Email = email, Password = request.Password });
+
+        // Assert
+        userInDb.Should().NotBeNull();
+
+        firstHttpResponse.StatusCode.Should().Be(HttpStatusCode.OK);
+        secondHttpResponse.StatusCode.Should().Be(HttpStatusCode.OK);
+    }
+
+
+    [Fact]
+    public async Task Post_Request_ReturnsBadRequest_DueToExpiredResetToken()
+    {
+        // Arrange
+        const string email = "member@test.de";
+        var memberClient = await ClientFactory.CreateMemberClient(this._app, this.Context, email, "password");
+        var request = new R.Request
+        {
+            PasswordResetToken = "F01E0980-AB43-47E2-9300-DA69838E8797", Password = "newPassword"
+        };
+
+        var userInDb = await this.Context.Users.FirstOrDefaultAsync(user => user.Email == email);
```

Comment:
An die anderen angleichen
```suggestion
        var userInDb = this.Context.Users.First(user => user.Email == email);
```